Lib/test/test_pathlib/test_pathlib.py

```diff
@@ -1605,18 +1605,18 @@ def test_absolute_posix(self):
     )
     @needs_posix
     def test_open_mode(self):
-        old_mask = os.umask(0)
+        old_mask = os.umask(2)
```

Comment:
I would write it as `0o002`. And please add a short explanation comment.

Reply:
Done!